learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -236,13 +246,17 @@ resource function get filter(string isoCode) returns CovidData? {
 }
 ```
 
-In the above resource method, the `filter` field is defined in the root `Query` type. Since this field has an
-input parameter `isoCode`, you have to add an input parameter to the resource method. This method returns the
-corresponding data for the given `isoCode` if such data is available in our data set, otherwise, it returns `null`.
+In this code:
+- The `filter` field is defined in the root `query` type. Since this field has an input parameter `isoCode`, you have to add an input parameter to the resource method. 
```

Comment:
Please check all the occurrences. The `Query` is a predefined type that should start with a capital letter.

```suggestion
- The `filter` field is defined in the root `Query` type. Since this field has an input parameter `isoCode`, you have to add an input parameter to the resource method. 
```